docs/project_defn.rst

```diff
@@ -158,6 +159,7 @@ Additional Reporting Measures
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 Include the ``reporting_measures`` key along with a list of reporting measure names to apply additional reporting measures (that require no arguments) to the workflow.
 Any columns reported by these additional measures will be appended to the results csv.
+Note: For upgrade runs, do not add ``ApplyUpgrade`` to the list of reporting measures, doing so will cause run to fail prematurely. ApplyUpgrade is applied automatically when the ``upgrades`` key is supplied.
```

Comment:
@lixiliu Put second instance of "ApplyUpgrade" between "``".

Reply:
@joseph-robertson done